### PR TITLE
Lab #4

### PR DESCRIPTION
**Pull request by WilfredoN**

### CODE REVIEW

File: lab_04/main.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include <cmath>
+#include <iostream>
+
+float calculate_float(float a, float b, float c, float d, float e, float f) {
+  return (float)(std::powf(a, 2.0) + b * c - d) * (e - (b / c) + a * f) -
```
Comment (volnsav) [inline lab_04/main.cpp]: You don't have to use type casting here.

File: lab_04/main.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+         (c + a * d) + ((b * e) / (c - d * f));
+}
+
+int calculate_integer(int a, int b, int c, int d, int e, int f) {
+  return (int)(std::pow(a, 2) + b * c - d) * (e - (b / c) + a * f) -
```
Comment (volnsav) [inline lab_04/main.cpp]: You don't have to use type casting here.

File: lab_04/main.cpp [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+         (c + a * d) + ((b * e) / (c - d * f));
+}
+
+int main() {
+  int int_a1 = 2, int_b1 = 3, int_c1 = 4, int_d1 = 1, int_e1 = 6, int_f1 = 2;
```
Comment (volnsav) [inline lab_04/main.cpp]: This is not allowed, please use one declaration per line.

File: lab_04/main.cpp [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  int int_a2 = 5, int_b2 = 7, int_c2 = 8, int_d2 = 3, int_e2 = 4, int_f2 = 1;
+
+  std::cout << "Integer Calculation 1: "
```
Comment (volnsav) [inline lab_04/main.cpp]: Please use printf/scanf in the fall semester.

File: lab_04/main.cpp
Comment (volnsav) [inline lab_04/main.cpp:1]: Please add an analysis of the given expressions. How can we verify the results?

**Review by volnsav — COMMENTED** [review]
Please address comments.

File: lab_04/main.cpp
Comment (capynq) [reply lab_04/main.cpp:1]: should this be in the md format or something else?

Comment (volnsav) [reply lab_04/main.cpp:1]: md or comments in main.c

**Comment by volnsav** [comment]
Approved.

**Pull request merged**